features-sdk/step_definitions/security-steps.js

```diff
@@ -109,3 +110,8 @@ Given('The role {string} should match:', async function (roleId, dataTable) {
     should(actions).match(role.controllers[controller].actions);
   }
 });
+
+Then('I am able to mGet {int} users with the following ids:', async function (count, dataTable) {
```

Comment:
The `count` is unused here. Either check the number of returned user or do not add another variable